### PR TITLE
Test: add some unittest to addon cache

### PR DESCRIPTION
**Pull request by JarHMJ**
### Description of your changes

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Comment by JarHMJ** [comment]
please review @barnettZQG @wonderflow @leejanee

File: pkg/addon/cache_test.go [hunk 1]
```diff
@@ -31,3 +31,63 @@ func TestPutVersionedUIData2cache(t *testing.T) {
 	assert.NotEmpty(t, u.versionedUIData["helm-repo"]["fluxcd-1.0.0"])
 	assert.Equal(t, u.versionedUIData["helm-repo"]["fluxcd-1.0.0"].Name, "fluxcd")
 }
+
+func TestPutAddonUIData2Cache(t *testing.T) {
```
Comment (barnettZQG) [inline pkg/addon/cache_test.go:35]: How is this different from `TestPutVersionedUIData2cache`?

Comment (JarHMJ) [reply pkg/addon/cache_test.go:35]: These test different func in cache.go

**Review by wonderflow — APPROVED** [review]
good job, thanks!

**Pull request merged**